frontend/packages/core/src/stories/radio-group.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import type { Meta } from "@storybook/react";
+
+import type { RadioGroupProps } from "../Input/radio-group";
+import { RadioGroup } from "../Input/radio-group";
+
+export default {
+  title: "Core/Input/RadioGroup",
+  component: RadioGroup,
+  argTypes: {
+    onChange: { action: "onChange event" },
+  },
+} as Meta;
+
+const Template = (props: RadioGroupProps) => <RadioGroup {...props} />;
+
+export const Primary = Template.bind({});
+Primary.args = {
```

Comment:
You'll probably want to add argTypes to this as well to denote that the defaultOption prop should be selected from a list instead of a string input.

Reply:
added